src/mgr/ActionParams.cc

```diff
@@ -20,11 +20,13 @@ Mgr::ActionParams::ActionParams(): httpMethod(Http::METHOD_NONE)
 
 Mgr::ActionParams::ActionParams(const Ipc::TypedMsgHdr &msg)
 {
-    msg.getString(httpUri);
-
-    String method;
+    SBuf method;
     msg.getString(method);
-    httpMethod.HttpRequestMethodXXX(method.termedBuf());
+    httpMethod = HttpRequestMethod(method);
+
+    SBuf uri;
+    msg.getString(uri);
```

Comment:
Please load the URI first, as the official code does, as detailed in the change request for Mgr::ActionParams::pack().